tests/unit/models/generation/test_vllm_generation.py

```
@@ -1316,6 +1316,93 @@ def _standardize(d: dict) -> dict:
+    body = dict(
+        model=generation_config["model_name"],
+        messages=[
+            {"role": "user", "content": "count to 5"},
+        ],
+        temperature=generation_config["temperature"],
+        top_p=generation_config["top_p"],
+        logprobs=True,
+        return_tokens_as_token_ids=True,
+        max_tokens=1,
+    )
+    response = requests.post(url=f"{final_urls[0]}/chat/completions", json=body)
```

Ruff (0.15.21)

[error] Probable use of `requests` call without timeout

(S113)